pkg/supportbundle/spec.go

```diff
@@ -144,7 +144,12 @@ func CreateRenderedSpec(app apptypes.AppType, sequence int64, kotsKinds *kotsuti
 	renderedSpec = b.Bytes()
 
 	secretName := GetSpecSecretName(app.GetSlug())
-	existingSecret, err := clientset.CoreV1().Secrets(util.PodNamespace).Get(context.TODO(), secretName, metav1.GetOptions{})
+	_, err = AddLabelExistingSpecSecret(clientset, util.PodNamespace, secretName, kotstypes.GetTroubleshootLabels())
```

Comment:
would you mind adding a comment about why this functions is needed? (e.g. to add the troubleshoot labels to existing secrets that were created in older versions of kots)

Reply:
added a comment